### PR TITLE
support tri-backquote style code block

### PR DESCRIPTION
**Pull request by drunkdream**
Code block will wrapped by tri-backquote when `config.BACKQUOTE_CODE = True`, which default is `False`.
For example:

    ```python
    print('hello')
    ```

### CODE REVIEW

File: test/backquote_code.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<p>before pre</p>
+<pre><code class="python">print("hello")</code>
```
Comment (Alir3z4) [inline test/backquote_code.html]: Is it normal or sort of standard to use `class` attribute to mark the language of the code ?
Any example that I can find on internet that follow such rule ?

Comment (drunkdream) [reply test/backquote_code.html]: I find some websites in [this link](
https://publicwww.com/websites/%22%3Cpre%3E%3Ccode+class%3D%5C%22python%5C%22%22/)

Comment (jdufresne) [reply test/backquote_code.html]: Hmm. I'm most used to seeing this convention on GitHub and it doesn't appear to follow this convention. For example, inspect the HTML of the following.

```python
print("hello")
```

The HTML is:

```
<div class="highlight highlight-source-python"><pre><span class="pl-c1">print</span>(<span class="pl-s"><span class="pl-pds">"</span>hello<span class="pl-pds">"</span></span>)</pre></div>
```

I like the idea of the backquotes, but perhaps we should leave out the language name until we can identify a more consistent convention.

**Comment by Alir3z4** [comment]
Closing this as being inactive and not addressing the comments.
Please feel free to reopen or make a new pull request to update the code.

**Pull request closed without merging**